Docker/publish/tag_git/tag_git.sh

```diff
@@ -12,7 +12,7 @@ done
 
 if [ -z "$WIKIBASE_BRANCH_NAME" ] || \
 [ -z "$RELEASE_VERSION" ] || \
-[ -z "$QUERYSERVICE_UI_COMMIT_HASH" ] || \
+[ -z "$WDQS_UI_COMMIT_HASH" ] || \
```

Comment:
If we want to make this change we need to make corresponding changes to .env and the versions/* files I think.

Reply:
In this case WDQS_UI_COMMIT_HASH is read (if present) from build_metadata_wdqs_ui.env which is written at checkout which is currently set by QUERYSERVICE_UI_COMMIT_HASH. 

I think these intentionally became two different variables as we want the build_metadata file to determine what commit is tagged. However looking at it again i realize the naming should be updated to reflect that one is used for building and one is used as an artifact later from the build.

Reply:
added the METADATA_ prefix to make this difference more clear.